sycl/test/basic_tests/buffer/buffer_for_not_device_copyable.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl -fsyntax-only -Xclang -verify %s -Xclang -verify-ignore-unexpected=note,warning
+
+#include <sycl/sycl.hpp>
+#include <string>
+
+using namespace cl::sycl;
```

Comment:
```suggestion
using namespace sycl;
```

Reply:
@alexbatashev Thank you, the namespace is changed to `sycl::`.